### PR TITLE
Created SMSStringEvent

### PR DESCRIPTION
**Pull request by turcato-niccolo**
Quick pull request to keep everyone up to date, I'm setting up the parts for the demo

### CODE REVIEW

File: app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class SMSStringEventTest {
+    private static final String TEST_NUMBER = "+390000111222";
+    private static final String TEST_DESCRIPTION = "Name: description";
+    private static final double LAT = 13;
```
Comment (CremaLuca) [inline app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java]: Codacy Issue found: [Avoid unused private fields such as 'LAT'.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 3
           

Complexity increasing per file
==============================
- app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java  1
- app/src/main/java/com/eis/geoCalendar/network/SMS/SMSStringEvent.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395&bid=16698002)

File: app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    private static final double LON = 14;
+
+    private GPSPosition testPosition;
+    private SMSStringEvent testEvent;
+    private SMSNetworkEventUser testUser;
+    private SMSPeer testPeer;
```
Comment (CremaLuca) [inline app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java]: Codacy Issue found: [Perhaps 'testPeer' could be replaced by a local variable.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395)

File: app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class SMSStringEventTest {
+    private static final String TEST_NUMBER = "+390000111222";
+    private static final String TEST_DESCRIPTION = "Name: description";
+    private static final double LAT = 13;
+    private static final double LON = 14;
```
Comment (CremaLuca) [inline app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java]: Codacy Issue found: [Avoid unused private fields such as 'LON'.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395)

**Review by tonin-alessandra — APPROVED** [review]

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java  1
- app/src/main/java/com/eis/geoCalendar/network/SMS/SMSStringEvent.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395&bid=16698002)

File: app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class SMSStringEventTest {
+    private static final String TEST_NUMBER = "+390000111222";
+    private static final String TEST_DESCRIPTION = "Name: description";
+    private static final SMSPeer testPeer = new SMSPeer(TEST_NUMBER);
```
Comment (CremaLuca) [inline app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java]: Codacy Issue found: [Name 'testPeer' must match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395)

**Comment by CremaLuca** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- app/src/test/java/com/eis/geoCalendar/network/SMS/SMSStringEventTest.java  1
- app/src/main/java/com/eis/geoCalendar/network/SMS/SMSStringEvent.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5075395&bid=16698002)

**Pull request merged**